app/controllers/shipit/deploys_controller.rb

```diff
@@ -40,6 +41,12 @@ def revert
       redirect_to rollback_stack_deploy_path(@stack, previous_deploy)
     end
 
+    def short_until_commit(task)
```

Comment:
Nit: `commit` is a real model, this returns the sha of a matched commit as a string so `short_commit_sha`?